### PR TITLE
Ypw

### PR DESCRIPTION
**Pull request by Yanping123DFEF**
Could you please check the documentation for function mk_reactable and tidy_ae_table?

### CODE REVIEW

**Comment by elong0527** [comment]
@wangben718, please review. Thanks!

File: R/mk_reactable.R [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+#' @param searchable make the entire table searchable using searchable
+#' @param defaultPageSize The page size options be customized 10 through pageSizeOptions
+#' @param borderless  customize table styling using Borderless
+#' @param striped customize table styling using Stiped
+#' @param highlight customize table styling using highlight
 #'
+#' @return an interactive data table with filtering,searching,pagination and custom table styling
+#' using Bordered + striped + highlighting
 #' @export
+#' @examples
```
Comment (wangben718) [inline R/mk_reactable.R:17]: Please make sure this example could be ran properly. Also, we should have an example for basic use as well, such as mk_reactable(iris).

File: R/tidy_ae_table.R [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
-#' library(dplyr)
-#' library(tidyr)
-#' db <- adsl %>% rename(TRTA = TRT01A)
-#' tidy_ae_table(population_from  = db,
-#'               observation_from = adae,
-#'               treatment_var = "TRTA",
-#'               treatment_order = c("MK9999" = "Xanomeline High Dose", "Placebo" = "Placebo"))
 #'
-#' @export
+#' @param population_from data sources
```
Comment (wangben718) [inline R/tidy_ae_table.R:7]: A data frame to obtain population level variables.

File: R/tidy_ae_table.R [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+#' @param observation_from data sources
```
Comment (wangben718) [inline R/tidy_ae_table.R:8]: A data frame to obtain observation level variables.

File: R/tidy_ae_table.R [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
+#' @param population_where Select the rows which is "Y" in column "ITTFL"
```
Comment (wangben718) [inline R/tidy_ae_table.R:9]: A character string to define the criteria to select analysis population.

File: R/tidy_ae_table.R [hunk 5]
```diff
@@ -1,17 +1,28 @@
...
+#' @param observation_where A character string to define the criteria to select analysis observation.
+#' @param treatment_var A character string to define the variable of new column called "treatment"
+#' @param treatment_order To customize the sort order
+#' @param ae_var Select the Desired variable
+#' @param stratum_var A character string to define the variable of baseline stratum in 'population_from'.Only one 'stratum_var' is allowed.
+#' @param listing_var Tells the function which column of the table to be selected as the final output
 #'
+#' @return an data frame
+#' @export
+#' @example
```
Comment (wangben718) [inline R/tidy_ae_table.R:19]: I think it should be @examples

File: R/tidy_ae_table.R [hunk 6]
```diff
@@ -1,17 +1,28 @@
...
-#'               treatment_var = "TRTA",
-#'               treatment_order = c("MK9999" = "Xanomeline High Dose", "Placebo" = "Placebo"))
 #'
-#' @export
+#' @param population_from data sources
+#' @param observation_from data sources
+#' @param population_where Select the rows which is "Y" in column "ITTFL"
+#' @param observation_where A character string to define the criteria to select analysis observation.
+#' @param treatment_var A character string to define the variable of new column called "treatment"
+#' @param treatment_order To customize the sort order
```
Comment (wangben718) [inline R/tidy_ae_table.R:12]: For line 12, 13, 15, I think it should have the same kind of expression as line 10.
@param treatment_order A character vector to define the treatment display order and label.
@param ae_var A character string to define the variable of new column called ae"
@param listing_var ...

File: R/tidy_ae_table.R [hunk 7]
```diff
@@ -1,17 +1,28 @@
...
+#' @param ae_var Select the Desired variable
+#' @param stratum_var A character string to define the variable of baseline stratum in 'population_from'.Only one 'stratum_var' is allowed.
+#' @param listing_var Tells the function which column of the table to be selected as the final output
 #'
+#' @return an data frame
```
Comment (wangben718) [inline R/tidy_ae_table.R:17]: Return a standard adverse event data frame

**Comment by wangben718** [comment]
> Could you please check the documentation for function mk_reactable and tidy_ae_table?
I have reviewed and provided some comments. Please check and let me know if there is any issue.
@Yanping123DFEF @elong0527

**Comment by Yanping123DFEF** [comment]
sure，Thank you！

best，
Yanping

**Review by elong0527 — APPROVED** [review]

**Pull request merged**